### ISSUE TITLE
flush() on unchanged entities collection result in rewriting collection with same data

### ISSUE THREAD

**Issue by theUm** [graphaware/neo4j-php-ogm#101]
here`s code:
![code](https://cloud.githubusercontent.com/assets/3176889/23371647/7df58406-fd21-11e6-98fd-5a13d1fe0e41.png)

here`s result, repo found 15 entities and in UnitOfWork they marked under `update_nodes` tag:
![view](https://cloud.githubusercontent.com/assets/3176889/23371652/81f023a4-fd21-11e6-9dd7-bf821b4fc78e.png)

**Comment by ikwattro** [comment]
Yes this is known, it is called super consistency :) rc2 fix

**Comment by Sraime** [comment]
I have the same problem. It slows considerably the performance in my case because one node can have around 500 relations. Will it be fixed for the RC6? :/

**Comment by ikwattro** [comment]
Yes next release will be focused on stability of the UOW, I think it will be the last RC. So maybe it will go direct in GA release by the end of next week.

**Comment by Sraime** [comment]
Hi ! For when is planned the next release? I will try to fix it by myself and provide a PR as soon as possible

**Comment by ikwattro** [comment]
I think next release will be by the end of the month. of course PR welcome :)
